Eclipse

Rampoch opened the pull request:
Возвращение обновленного Эклипса.

<details>
<summary>Ченджлог</summary>

```yml
🆑
maptweak: Добавление карты Eclipse с обновлениями.
maptweak: Переделан дормиторий.
maptweak: Переделана оранжерея.
maptweak: Перенос чекпоинта Чарли и обновление техтоннелей рядом.
maptweak: Перенос своих зон в отдельный файл.
maptweak: Обновлен отдел суперматерии.
maptweak: Обновлен отдел сингулярности, намеренно усложнена сборка ускорителя и добавлена возможность охлаждения коллекторов которые во время тестов выдерживали нагрев от радиации, но требуется дополнительный фитбек от продвинутых инженеров. Новичкам запускать не рекомендуется.
maptweak: Добавлен гетто-дормиторий.
maptweak: Добавлен генератор гравитации и крыло продвинутого техобеспечения.
maptweak: Расширены техтоннели рнд.
maptweak: Добавлен гетто-ии.
maptweak: Добавлен офис адвоката и театр.
maptweak: Добавлена постоянная тюрьма со своим астероидом.
maptweak: Обновлены предметы в отделах в соответствии с нововведениями билда.
maptweak: Починены и обновлены шаттлы.
maptweak: Прочие мелкие фиксы и обновления карты.
/🆑
```

</details>

- [x] Pull Request полностью завершен, мне не нужна помощь чтобы его закончить.
- [x] Я внимательно прочитал все свои изменения и багов в них не нашел.
- [x] Я запускал сервер со своими изменениями локально и все протестировал.
- [x] Я ознакомился c [Guide to Contribute](https://github.com/ChaoticOnyx/OnyxBay/blob/dev/docs/contributing.md).

Rampoch committed:
Eclipse

Возвращение Эклипса:
-переделан дормиторий
-переделана оранжерея
-перенос своих зон в отдельный файл
-обновлены отделы материи и синги
-добавлен гетто-дормиторий
-добавлен генератор гравитации и крыло продвинутого техобеспечения
-расширены техтоннели рнд
-добавлен гетто-ии
-добавлен офис адвоката и театр
-добавлена постоянная тюрьма со своим астероидом
-обновлены предметы в отделах в соответствии с нововведениями билда
-починены и обновлены шаттлы
-прочие мелкие фиксы и обновления карты

Rampoch committed:
Eclipse

Подключение файлов Эклипса.

Rampoch committed:
Eclipse

Мерж с актуальной версией.

Zert0X commented on maps/eclipse/eclipse_levels.dm:
```diff
@@ -0,0 +1,36 @@
+/datum/space_level/eclipse_1
+	path = 'eclipse-1.dmm'
+	traits = list(
+		ZTRAIT_STATION,
+		ZTRAIT_CONTACT
+	)
+	travel_chance = 5
+
+/datum/space_level/eclipse_1/generate(z)
+	// Create the mining Z-level.
+	new /datum/random_map/automata/cave_system(null, 1, 1, z, 255, 255)
+	new /datum/random_map/automata/cave_system/air(null, 1, 1, z, 255, 255)
+	// Create the mining ore distribution map.
+	new /datum/random_map/noise/ore(null, 1, 1, z, 255, 255)
+
+/datum/space_level/eclipse_3
+	path = 'eclipse-3.dmm'
+	traits = list(
+		ZTRAIT_CENTCOM,
+		ZTRAIT_CONTACT,
+		ZTRAIT_SEALED
+	)
+
+/datum/space_level/eclipse_6
+	path = 'eclipse-6.dmm'
+	travel_chance = 10
+	traits = list(
+		ZTRAIT_CONTACT
+	)
+
+/datum/space_level/eclipse_6/generate(z)
+	// Create the mining Z-level.
+	new /datum/random_map/automata/cave_system(null, 1, 1, z, 255, 255)
+	new /datum/random_map/automata/cave_system/air(null, 1, 1, z, 255, 255)
+	// Create the mining ore distribution map.
+	new /datum/random_map/noise/ore(null, 1, 1, z, 255, 255)
```
Где эклипсы 4 и 5

Rampoch replied:
Пока отпали за ненадобностью. Это были спутник связи и заброшенный корабль, которые перенесли в стандартно запускаемый список карт. Названия пускай на память останутся.

ArmolitskiyCarpoeb commented:
Все отлично, не вижу никаких проблем. Весьма качественный и логично написанный код.

Rampoch committed:
Eclipse

-Замена старых окон на новые фреймы по техническим вопросам.
-Замена одинарных затворок на двойные в коридорах.
-Мелкофиксы бластдоров.

Rampoch committed:
Eclipse

Мелкофикс тестовых оконных фреймов.

Rampoch committed:
Eclipse

Легкие улучшения тюремного мини-астероида, пока аппрувщики игнорят

Rampoch committed:
Eclipse

Фикс заборки в чекпоинте браво.

Rampoch committed:
Eclipse

-Расширен кабинет адвоката.
-Пофикшены некоторые доступы.
-Улучшения декора станции.

Rampoch committed:
Eclipse

Различные мелкие улучшения декора.
